images-list

```diff
@@ -206,7 +206,9 @@ gcr.io/google_containers/k8s-dns-sidecar rancher/mirrored-k8s-dns-sidecar 1.15.1
 gcr.io/google_containers/pause rancher/mirrored-pause 3.1
 gcr.io/google_containers/pause rancher/mirrored-pause 3.2
 gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.5.0
+gcr.io/kubebuilder/kube-rbac-proxy rancher/kube-rbac-proxy v0.14.0
```

Comment:
do we need to keep mirroring new tags to the old un-mirrored repo? Can we update to the mirrored repo when switching to the new tag?

Reply:
Probably not, but I just assumed if both were included they were both in use and would need the update. Unless we have some charts referring to `rancher/kube-rbac-proxy:latest` that would need to have the newest versions, we can stop.

From the blame @rawmind0 added the original unmirrored image and @rmweir added the mirrored image. Maybe they have more context for why we have both in the image list / if we need to keep them updated.

Reply:
@brandond are you ok with only bumping the mirrored images moving forward? looks like thats what we do for the `rancher/bats-bats` and `rancher/mirrored-bats-bats`

Reply:
yeah I think we should stop bumping tags for images without the mirrored- prefix.

Reply:
sick, fixed